src/js_native_api_v8.h

```diff
@@ -2,7 +2,7 @@
 #define SRC_JS_NATIVE_API_V8_H_
 
 #include <string.h>
-#include "js_native_api_types.h"
+#include "js_native_api.h"
```

Comment:
Moving this include to after `js_native_api_v8_internals.h` removes the warning

Reply:
I guess another property of `js_native_api_v8_internals.h` may be that it defines `NAPI_VERSION`. Could you maybe add a comment to the first paragraph of the comment in `js_native_api_v8_internals.h`to the effect of "This file may, at its option, define `NAPI_VERSION` and set it to the version of N-API to be exposed." In the second paragraph could you maybe add something like "node_internals.h, included below, also defines NAPI_VERSION".

Reply:
I will add the comment.